src/test/java/com/time_management/controllers/ReportControllerTest.java

```diff
@@ -1,4 +1,42 @@
 package com.time_management.controllers;
 
-public class ReportControllerTest {
+import com.time_management.app.dtos.reports.ReportResponseDTO;
+import com.time_management.app.ports.ReportService;
+import com.time_management.infra.input.controllers.ReportController;
+import org.instancio.Instancio;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.*;
+
+class ReportControllerTest {
+
+    @Mock
+    private ReportService reportService;
+
+    @InjectMocks
+    private ReportController reportController;
+
+    public ReportControllerTest() {
+        MockitoAnnotations.openMocks(this);
```

Comment:
A inicialização dos mocks (MockitoAnnotations.openMocks(this)) está sendo feita no construtor da classe de teste. Embora isso funcione, é mais comum e recomendado usar a anotação @BeforeEach do JUnit para inicializar os mocks antes de cada teste. Isso garante que os mocks sejam reinicializados para cada teste, evitando possíveis vazamentos de estado entre testes.
```suggestion
@BeforeEach
void setUp() {
    MockitoAnnotations.openMocks(this);
}
```